### PR TITLE
fix(#12403): áéíóú ÁÉÍÓÚ characters not allowed in glossary, glossary term, classification and tag names

### LINKED ISSUE #12403: áéíóú ÁÉÍÓÚ characters not allowed in glossary, glossary term, classification and tag names
**Affected module**
UI

**Describe the bug**
In  1.1.0 version á é í ó ú characters are not allowed now in glossary terms name. In spanish lang these characters are common and day to day used. 

**To Reproduce**
Reproduce with the UI for:

- new glossary
- new glossary term
- new classification
- new tag
.

Screenshots or steps to reproduce
image

image

image

**Expected behavior**
As in previous versions allow latin characters like: á é í ó ú in name, by UI and API

**Version:**
OM 1.1.0
python 3.8

**Additional context**

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

Fixes #12403 

I worked on issue #12403 

https://github.com/open-metadata/OpenMetadata/assets/59080942/4b937c1f-71b1-4ddd-9311-31f19a6eef3a

#
### Type of change:

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] My PR title is `Fixes <issue-number>: <short explanation>`
- [x] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/constants/regex.constants.test.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    expect(ENTITY_NAME_REGEX.test('HelloWorld123!')).toEqual(false);
+
+    // Contains letters, numbers, and @ special characters.
+    expect(ENTITY_NAME_REGEX.test('HelloWorld123@')).toEqual(false);
+
+    // Contains letters, numbers, and * special characters.
+    expect(ENTITY_NAME_REGEX.test('HelloWorld123*')).toEqual(false);
+
+    // Contains letters, numbers, and special characters.
+    expect(ENTITY_NAME_REGEX.test('!@#$%^&*()')).toEqual(false);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/constants/regex.constants.test.ts:85]: % will be allowed for name, Let's confirm this

Comment (harshach) [reply openmetadata-ui/src/main/resources/ui/src/constants/regex.constants.test.ts:85]: can we add "." as well

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/constants/regex.constants.test.ts:85]: @harshach , "." is already there, the above one is for the test case.

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/constants/regex.constants.test.ts:85]: @chirag-madlani , for now we are just allowing % in glossary and glossaryTerms name, ref : https://github.com/open-metadata/OpenMetadata/pull/12382

I think that we can handle it separately after [this](https://github.com/open-metadata/OpenMetadata/pull/12382) PR got merged.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**